### ISSUE TITLE
[Feature Request] Add Tab Order configuration to Layout & Styles

### ISSUE THREAD

**Issue by TonyC5** [wpninjas/ninja-forms#3167]
- **What actions are you taking?**
  - Trying to customize the tab order for a multi-column form created with Layout & Styles (L&S)
- **What are you expecting to occur?**
  - I expected a tab order configuration option, but there isn't one.
- **What is occurring instead?**
  - Without custom tab ordering, <Tab> and <Enter> advance the cursor down each column of `nf-cells`, which for my particular multi-column form is counter-intuitive.  Each row of related data needs to be completed before advancing to the next row, so my users expect <Tab> and <Enter> to advanced to the next column in the row (not to the next row in the column).

**In its simplest form, this feature could be a single option to advance by row, or advance by column.  There doesn't need to be a taborder number for every field.**

I understand that this feature request has been rejected in the past, because of the perceived complexity / confusion for the form designer.  The decision to incorporate this feature into your product should be based on the form users experience, not the form designer's experience.  Tab order is a fundamental behavior of forms that is expected by users.

**Comment by TonyC5** [comment]
Zachary Skaggs was kind enough to point me in a new direction in the Slack forum - thanks, Zachary!  While I was able to "deconstruct" my form into component rows, this was tedious.  The advantage of creating a single row first and then duplicating the row (duplicating one column at a time) is that the columns in the form are automatically aligned with the same width.  Creating each row separately requires manual alignment of the columns.  Now, any changes to my column widths need to be performed one at a time on each row, making maintenance of the form more tedious.

The most intuitive way to create a multicolumn form is to create columns that share a single width such that adjustment of the column width affects all rows together.  While my new layout achieves the desired tab ordering, it is not ideal for editing and maintainability.

A taborder configuration option is still necessary.
